packages/lesswrong/server/reviewVoteUpdate.ts

```diff
@@ -388,3 +397,54 @@ export const createReviewWinners = async () => {
     return createReviewWinner(post, idx, category, adminContext)
   }))
 }
+
+// If you made any mistakes with the rank-order of the ReviewWinners (i.e. because you decided
+// to remove a post from the list), run this function to fix the rank orders.
+//
+// This is necessary because the reviewRanking has enforced uniqueness for rank+year, and
+// you can't edit an individual
+//
+// (This is similar but not identical to the updateCuratedOrder function in ReviewWinnersRepo,
+// which handles a similar case for the curatedOrder field, although only one post at a time)
+export const updateReviewWinnerRankings = async (year: number) => {
+  const adminContext = createAdminContext();
+  
+  const [reviewWinners, posts] = await Promise.all([
+    ReviewWinners.find({reviewYear: year}).fetch(),
+    Posts.find({
+      _id: { $in: (await ReviewWinners.find({reviewYear: year}, {projection: {postId: 1}}).fetch())
```

Comment:
if you're fetching posts based on the review winners, it doesn't make sense to try to parallelize the two queries; you're just making an extra db call for no reason

Reply:
(not a blocker since I assume this is a rarely-run script, but)

Reply:
Ah gotcha. I went and fixed it.